packages/preferences/src/components/preferences-modal-tabs/index.js

```diff
@@ -146,6 +146,7 @@ export default function PreferencesModalTabs( { sections } ) {
 										gap="6"
 									>
 										<Navigator.BackButton
+											size="compact"
```

Comment:
In the block editor, open the Preferences modal in a narrow viewport.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/f6ed8dd6-e14f-49ac-a6d9-f69a63dec007" alt="Back button in Preferences modal, before" width="669">|<img src="https://github.com/user-attachments/assets/e08798bb-db5b-4400-a275-644b1bd4771c" alt="Back button in Preferences modal, after" width="670">|